forc/src/cli/commands/test.rs

```diff
@@ -142,6 +147,21 @@ fn print_tested_pkg(pkg: &TestedPackage, test_print_opts: &TestPrintOpts) -> For
         // If logs are enabled, print them.
         if test_print_opts.print_logs {
             let logs = &test.logs;
+            if test_print_opts.decode_logs {
+                for log in logs {
+                    if let Receipt::LogData {
+                        rb,
+                        data: Some(data),
+                        ..
+                    } = log
+                    {
+                        let decoded_log_data = decode_log_data(*rb, data, &pkg.built.program_abi)?;
+                        let var_value = decoded_log_data.value;
+                        info!("Decoded log value: {}, log rb: {}", var_value, rb);
```

Comment:
Should we just print the value without anything before it? 

What is `rb`, and how would it be useful?

Reply:
Since we do not have variable names, there is no way for users to differentiate between different logs. It is coming from the raw log and incremented by one with each log, they would have to count number of logs in the output to match the log statements in the code. We can remove it but i was planning to remove it once we have var names implemented.

Reply:
Gotcha. I'm ok with this as a temporary solution until we have var names.